server/depositTarget/queries.ts

```diff
@@ -1,14 +1,21 @@
 import * as types from 'types';
 import { DepositTarget } from 'server/models';
 
+/**
+ * Get the primary deposit target for a Community. The primary deposit target is currently
+ * the first one created for the Community. Eventually, we will allow users to select the
+ * primary deposit target when a Community has more than one.
+ */
 export const getCommunityDepositTarget = (
 	communityId: string,
-	service: types.DepositTarget['service'] = 'crossref',
+	includeCredentials = false,
 ): Promise<types.Maybe<types.DepositTarget>> => {
 	return DepositTarget.findOne({
 		where: {
 			communityId,
-			service,
+		},
+		attributes: {
+			exclude: includeCredentials ? [] : ['username', 'password', 'passwordInitVec'],
```

Comment:
A quick fix to ensure we don't send deposit target credentials to browsers viewing dashboard settings pages.